### ISSUE TITLE
Clinical dataset validation

### ISSUE THREAD

**Issue by bhillmann** [knights-lab/analysis_SHOGUN#8]
- [x] Run deep shotgun data with disease association (need to download/curate a couple of datasets) to show same findings as deep shotgun

@RRShieldsCutler Has some of these datasets

**Comment by bhillmann** [comment]
@RRShieldsCutler I added the folder below with the README for this task:

```
clinical_datasets/
```

How close is the final dataset to running?

I would like to use [strandex](https://github.com/mdshw5/strandex) to subsample these FASTQs after quality control and run them all with SHOGUN RefSeq and IMG.

**Comment by RRShieldsCutler** [comment]
Sorry just saw this (git only emailed me the text "clinical_datasets" and none of your questions).

I ran shizen without flash on the dataset, trim_l to 50. I downsampled the R1's to 500k reads. Also converted both the deep and shallow to fasta. All those files are in this directory:
/project/flatiron2/data/public_shotgun/karlsson2013/shizen_20161130

Yesterday I ran shogun on both the deep and shallow set using utree_abfvh. Results are located:
Original depth: /project/flatiron2/robin/results/shogun_analysis_karlsson/deep_shotgun
Downsample: /project/flatiron2/robin/results/shogun_analysis_karlsson/shallow/

From this dataset
http://www.nature.com/nature/journal/v498/n7452/full/nature12198.html

**Comment by RRShieldsCutler** [comment]
This dataset:
http://www.nature.com/nature/journal/v498/n7452/full/nature12198.html

**Comment by bhillmann** [comment]
Can you rerun the commands with the newest UTree versions?

**Comment by RRShieldsCutler** [comment]
Yup. Running currently, they should be done by the evening sometime. They'll be in the same directories as pasted above.

**Comment by RRShieldsCutler** [comment]
Both are finished, FYI. I confirmed that the confidence intervals are updated in the tsv files.
Original depth: /project/flatiron2/robin/results/shogun_analysis_karlsson/deep_shotgun
Downsample: /project/flatiron2/robin/results/shogun_analysis_karlsson/shallow/

**Comment by bhillmann** [comment]
Awesome, great work.

**Comment by bhillmann** [comment]
```
/project/flatiron2/data/public_shotgun/karlsson2013/map.txt
```

**Comment by RRShieldsCutler** [comment]
Currently re-running with the new complete-species utree. The deep (full reads) should finish tonight sometime here:
```
/project/flatiron2/robin/results/shogun_analysis_karlsson/161208_analysis/deep/shogun_utree_lca_out
```

The shallow (downsampled) reanalysis is complete and located here:
```
/project/flatiron2/robin/results/shogun_analysis_karlsson/161208_analysis/shallow/shogun_utree_lca_out
```
